src/k8s-extension/azext_k8s_extension/partner_extensions/DefaultExtension.py

```diff
@@ -50,6 +55,21 @@ def Create(
                 scope_namespace = ScopeNamespace(target_namespace=target_namespace)
                 ext_scope = Scope(namespace=scope_namespace, cluster=None)
 
+        plan = None
+        if plan_name is not None or plan_product is not None or plan_publisher is not None:
+            if plan_name is None:
+                raise ArgumentUsageError('Usage error: Missing valid plan name. To find the correct plan name please refer to the Marketplace portal under ‘Usage Information + Support.’ The Plan Id listed here will be the valid plan name required.')
+            if plan_product is None:
+                raise ArgumentUsageError('Usage error: Missing a valid product name. To find the correct plan product please refer to the Marketplace portal under ‘Usage Information + Support.’ The Product Id listed here will be the valid plan product required.')

```

Comment:
Is it Plan Product or Product Name?  Stick to the same term in help text as well

Reply:
It should be plan product. updated. thank you